Base/res/ladybird/about-pages/settings.html

```diff
@@ -74,10 +74,19 @@
                 padding: 15px 20px;
             }
 
+            .header-level-2 {
+                font-size: 16px;
+                font-weight: 500;
+            }
+
             .card-body {
                 padding: 20px;
             }
 
+            .body-level-2 {
+                padding: 0 20px;
```

Comment:
I think we want some top padding on this one, the body looks squished against the header to me:
![squish](https://github.com/user-attachments/assets/4ca663e3-5276-4b29-bf5b-64aff9d7e0e1)

I think we can actually just remove the `body-level-2` class, it looks less squished to me that way:
![after](https://github.com/user-attachments/assets/b56bb1c4-4d74-4978-b34a-1aef4497f89a)